lib/blocks/misc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = blockConfig => (processEnv, argv) => config => {
+    const defaultConf = {
+        ignored: /node_modules/,
+        stats: {
+            colors: true,
+            version: false,
+            timings: false,
+            assets: false,
+            chunks: false,
+            modules: false,
+            reasons: false,
+            children: false,
+            source: false,
+            errors: true,
+            errorDetails: true,
+            warnings: true
+        },
+        extensions: [".js", ".jsx"]
```

Comment:
I think it's weird to have that in `misc`, I think it could be logic to have it with the babel part. What do you think ?